modules/openy_features/openy_block/modules/openy_code_block/openy_code_block.info.yml

```diff
@@ -1,5 +1,6 @@
 name: 'OpenY Code Block'
 description: 'OpenY Code Block'
+package: OpenY
 type: module
 core: 8.x
 dependencies:
```

Comment:
Bump module version, please

Reply:
@podarok do you mean to set there the same version of the module as the current OpenY version, or use something independent?

Reply:
@podarok I have checked OpenY custom modules and features and they all not contain version... In this case I think better to create separated issue for all modules.